averbis/core/_rest_client.py

```diff
@@ -637,6 +695,12 @@ def __init__(self, project: "Project", identifier: str):
         self.project = project
         self.identifier = identifier
 
+    def __repr__(self):
+        """
+        A nicer string representation of the Pear object.
```

Comment:
Do we really need PyDoc for the `__repr__` method? I'd prefer to remove that. Python itself already defines what this methods does and we don't change anything about the meaning of the method.

Reply:
Alright, will change it.